ironfish/src/rpc/routes/multisig/createSigningCommitment.ts

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { roundOne } from '@ironfish/rust-nodejs'
+import * as yup from 'yup'
+import { ApiNamespace } from '../namespaces'
+import { routes } from '../router'
+import { RpcSigningCommitments, RpcSigningCommitmentsSchema } from './types'
+
+export type CreateSigningCommitmentRequest = {
+  keyPackage: string
+  seed: number
```

Comment:
Add TODO here to remove this once we are using deterministic nonces.

Reply:
done